app/screens/thread/thread.tsx

```diff
@@ -126,6 +127,9 @@ const Thread = ({
                             rootPost={rootPost!}
                         />
                     </View>
+                    <ScheduledPostIndicator
```

Comment:
0/5 should we add test for this?

Reply:
No initial test for this component. Can be taken up later.

Reply:
https://mattermost.atlassian.net/browse/MM-62286